### PR TITLE
MTE 5293 - trim zoolander

### PR DESCRIPTION
**Pull request by runrog**
I split this PR into 3 commits as it's a lot. Here's what it covers
- removes a lot of unused node modules
  - pify [promise library] (not used)
  - braces (not used)
  - clean-css (not used)
  - fs-extra (not used)
  - nan (not used)
  - request (not used)
  - semver (not used)
  - shortid (not used)
  - walk (not used)
  - concurrently (moved to dev dependency)
  - mozilla-fira-pack (moved to dev dependency)
  - node-sass (moved to dev dependency)

- automates icon documentation. It does this by automating the _harp.json global variables file. On `npm start`, a node script grabs the icons list from the fontello config, then adds that list as a global variable, then the icons page loops that and outputs all of our icons. (no more manually updating the icons page every time there's an icon update)

- removes old 0365 tables / zoolander page references.

- removes old resource center styling and old markup. no more references to "refresh", it's not just resource center.

- various documentation cleanup where page layouts were broken and links weren't valid.

### CODE REVIEW

File: dist/css/derek.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
...width:0}.panel>.table,.panel>.table-responsive>.table,.panel>.panel-collapse>.table{margin-bottom:0}.panel>.table caption,.panel>.table-responsive>.table caption,.panel>.panel-collapse>.table caption{padding-right:15px;padding-left:15px}.panel>.table:first-child,.panel>.table-responsive:first-child>.table:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>t
```
Comment (WidgetsBurritos) [inline dist/css/derek.css]: Let's not push any of the `dist` folder up.  The version patching will handle this for us.

**Review by WidgetsBurritos — CHANGES_REQUESTED** [review]

File: dist/css/derek.css (hunk 1 above)
Comment (runrog) [reply dist/css/derek.css]: done. not sure what happened there.

**Review by WidgetsBurritos — APPROVED** [review]
:+1: LGTM

**Comment by WidgetsBurritos** [comment]
[run ci]

**Pull request merged**